frontend/packages/console-app/src/components/console-operator/ConsolePluginManifestPage.tsx

```diff
@@ -14,10 +16,25 @@ export const ConsolePluginManifestPage: React.FC<PageComponentProps> = ({ obj })
     pluginName,
   ]);
 
+  const manifestJson = React.useMemo(() => {
+    return pluginManifest ? JSON.stringify(pluginManifest, null, 2) : '';
+  }, [pluginManifest]);
+
   return (
     <PaneBody>
       {pluginManifest ? (
-        <CopyToClipboard value={JSON.stringify(pluginManifest, null, 2)} />
+        <CodeEditor
+          value={manifestJson}
+          language={Language.json}
```

Comment:
If we want to make it extra clear this is read only we can also add a header

```suggestion
          language={Language.json}
          headerMainContent={t('console-app~console-extensions.json (read only)')}
```

<img width="1635" height="1010" alt="image" src="https://github.com/user-attachments/assets/0729c570-a334-4a52-a750-6038a7c3bbdc" />